glances/plugins/glances_fs.py

```diff
@@ -186,7 +186,8 @@ def update(self):
                         stats.append(fs_current)
 
         # Update the stats
-        self.stats = stats
+        seen = set()
+        self.stats = [seen.add(stat['mnt_point']) or stat for stat in stats if stat['mnt_point'] not in seen]
```

Comment:
@rhoriguchi 
LGTM

Could you just please mention the pull request number or give a one line description on why those two lines are needed in the fix?

Would be lot easier later on to understand why we need this list comprehension

Reply:
Is this good enough for you `# Create unique list filter by "mnt_point" and update the stats`?